src/applications/vaos/new-appointment/components/ClinicChoicePage.jsx

```diff
@@ -28,7 +28,9 @@ function getPageTitle(schema, typeOfCare) {
   const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
   let pageTitle = 'Clinic choice';
   if (schema?.properties.clinicId.enum.length === 2) {
-    pageTitle = `Make a ${typeOfCareLabel} appointment at your last clinic`;
+    pageTitle = `Make ${
+      vowelCheck(typeOfCareLabel) ? 'an' : 'a'
```

Comment:
Added @LanceGundersen's vowel check to the main header string, which had the same problem